language-support/ts/daml-ledger/index.test.ts

```diff
@@ -171,14 +175,14 @@ describe("streamSubmit", () => {
     expect(mockChange).not.toHaveBeenCalled();
   });
 
-  test("reconnect on close", async () => {
+  test("reconnect on server close", async () => {
     const reconnectThreshold = 200;
     const ledger = new Ledger({...mockOptions, reconnectThreshold: reconnectThreshold});
     const stream = ledger.streamQuery(Foo);
     stream.on("live", mockLive);
     stream.on("close", mockClose);
     mockInstance.serverSend({events: [], offset: '3'});
-    await new Promise(resolve => setTimeout(resolve, reconnectThreshold));
+    await new Promise(resolve => setTimeout(resolve, reconnectThreshold * 2));
```

Comment:
We don't _need_ to, I suppose: I've not seen this fail yet. Call it ~paranoia~ prudence: sleep instructions are always a bit fuzzy in their guarantees, so I prefer to sleep "significantly" longer than a timeout I'm testing for. These are milliseconds and we have like 4 lines of code here so it's cutting it really close otherwise.